File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
  public static void main(String[] args) throws IOException {

    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		StringTokenizer st = new StringTokenizer(br.readLine()," ");
    
		int hour = Integer.parseInt(st.nextToken());
		int minute = Integer.parseInt(st.nextToken());

		int addTime = Integer.parseInt(br.readLine());

		minute = hour*60 + minute + addTime;

		hour = minute / 60;
		minute = minute % 60;

		if (hour >= 24) {
			hour = hour - 24;
		}

		System.out.println(hour + " " + minute);

	}
}

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.kt
fun main() {
    var (A, B) = readLine()!!.split(" ").map { it.toInt() }
    var C = readLine()!!.toInt()

    B += C

    if (B >= 60) {
        A = A + B / 60
        B = B % 60
    }

    if (A >= 24) A -= 24

    print("$A $B")
}